docs/HowToBuild.html

```diff
@@ -437,70 +437,76 @@ <h2 class="categoryTitle">Testing Plugins</h2>
 </div>
 
 <div class="categoryBlock" id="DebuggingPlugins">
-        <h2 class="categoryTitle">Debugging Plugins</h2>
-        <div class="categoryDetailBlock">
-            <p class="categoryDetailBody">Panel-based FormIt plugins utilize two distinct JavaScript engines. One is in the panel displaying the HTML, and another engine which is built-in to FormIt for Windows. These JavaScript engines are actually in two different processes.</p>
-            <p class="categoryDetailBody"><b>FormIt Debugging</b></p>
-            <p class="categoryDetailBody">To debug in FormIt-side code (for toolbars or panels), you can enter the following to pop the desktop application's built-in JS debugger:</p>
-            <ul class="categoryDetailList">
-                <li class="categoryDetailCode"><span>debugger;</span></li>
-            </ul>
-            <p class="categoryDetailBody"><b>HTML Debugging</b></p>
-            <p class="categoryDetailBody">To debug HTML-side code (for panels only), including scripts and styling:</p>
-            <ul class="categoryDetailList">
-                <li><b>FormIt for Windows v17 and newer</b></li>
+    <h2 class="categoryTitle">Debugging Plugins</h2>
+    <div class="categoryDetailBlock">
+        Plugins can be debugged in either the Windows application, or the web application.
+
+        <ul class="categoryDetailList">
+            <li><a href="#DebuggingWindows">Debugging plugins on Windows (not recommended)</a></li>
+            <li><a href="#DebuggingWeb">Debugging plugins on web (recommended/easier option)</a></li>
+        </ul>
+    </div>
+</div>
+
+<div class="categoryBlock" id="DebuggingWindows">
+    <h2 class="categoryTitle">Debugging plugins on Windows</h2>
+
+    <p>Important note: A recent change requires that debugging plugins through the Windows app now requires a browser other than Chrome/Firefox. Opera is one of these browsers, and can be installed from https://www.opera.com/ . Using Opera, you will not be able to visualize your plugin, but you can at least debug your code.</p>
```

Comment:
Just to avoid redundancy, can you change this to "A recent change means that..."